core/trino-main/src/main/java/io/trino/execution/QueryManagerConfig.java

```diff
@@ -81,6 +81,7 @@ public QueryManagerConfig setScheduleSplitBatchSize(int scheduleSplitBatchSize)
     }
 
     @Min(1)
+    @Deprecated
```

Comment:
See [my other comment](https://github.com/trinodb/trino/pull/7394#issuecomment-805025699) about whether or not deprecation is the right idea here.